packages/ds-healthcare-gov/src/components/Header/Header.tsx

```diff
@@ -142,8 +142,9 @@ export const VARIATION_NAMES = {
  * [refer to its full documentation page](https://design.cms.gov/components/header/healthcare-header/?theme=healthcare).
  */
 export const Header = (props: HeaderProps) => {
-  const [openMenu, setOpenMenu] = useState(false);
   const isControlledMenu = props.isMenuOpen !== undefined && props.onMenuToggle !== undefined;
+  const [internalIsMenuOpenState, setInternalIsMenuOpenState] = useState(false);
+  const isMenuOpen = isControlledMenu ? props.isMenuOpen : internalIsMenuOpenState;
```

Comment:
This is just me cleaning up this info and making it consistent with some of our other controlled/uncontrolled components.